nemo/collections/llm/gpt/data/fine_tuning.py

```diff
@@ -212,7 +212,10 @@ def _extract_tokenizer_model_name(self) -> str:
             tokenizer_model_name = self.packed_sequence_specs.tokenizer_model_name
         elif isinstance(self.tokenizer, AutoTokenizer):
             name = self.tokenizer.tokenizer.name_or_path
-            if name.endswith("nemo_tokenizer"):
+            if name.endswith("context/nemo_tokenizer"):
```

Comment:
n00b question: how is the name set for the tokenizer?

Reply:
the tokenizer is restored here https://github.com/nvidia/NeMo/blob/9d857baf82ce163801459fa7cec71c1e38016b02/nemo/lightning/resume.py#L43
name is just the path